documentation/modules/con-securing-http-interface.adoc

```diff
@@ -6,11 +6,17 @@
 = Securing the HTTP Bridge HTTP interface
 
 [role="_abstract"]
-Authentication and encryption between HTTP clients and the HTTP Bridge is not supported directly by the HTTP Bridge.
-Requests sent from clients to the HTTP Bridge are sent without authentication or encryption.
-Requests must use HTTP rather than HTTPS.
+By default, requests sent from clients to the HTTP Bridge are sent without encryption therefore they must use HTTP.
```

Comment:
I would not put the sentence this way. I mean, the statement is from a client perspective saying that by default their requests are not encrypted (but how do you know? you are not in control of clients). We should put the sentence from a bridge perspective saying that by default the connection with client is not encrypted but it can be enabled. Hopefully you got what I mean :-)